flake.nix

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I'm not familiar with nix, but trust you here that this is fine. Maybe you could add a short entry in the readme (or a comment here in this file) what the purpose of this is?